# **2025_EVA_Quercus_Taxonomic_Diversity**

## This is the git repository for the publication:

**Fine-scale variation in soil properties promotes local taxonomic diversity of hybridizing oak species (_Quercus_ spp.)**


## Abstract

Although many tree species frequently hybridize and backcross, management decisions in forestry and nature conservation are usually concentrated on pure species. Therefore, understanding which environmental factors drive the distribution and admixture of tree species on a local stand scale is of great interest to support decision-making in the establishment and management of resilient forests. Here, we extensively sampled a mixed stand of hybridizing white oaks (*Quercus petraea* and *Q. pubescens*) near Lake Neuchâtel (Switzerland), where limestone and glacier moraine geologies coexist in proximity, to test whether micro-environmental conditions can predict taxonomic distribution and genetic admixture. We collected DNA from bud tissue, individual soil samples, and extracted high-resolution topographic data for 385 oak trees. We used 50 species-discriminatory single nucleotide polymorphism (SNP) markers to determine the taxonomic composition and admixture levels of individual trees and tested their association with micro-environmental conditions. We show that the trees' taxonomic distribution can be explained mainly by geographic position, soil pH, and potential rooting depth, a proxy for soil water availability. We found that admixed individuals tend to grow in habitats that are characteristic of the more drought-tolerant species *Q. pubescens* rather than in intermediate habitats. Using in situ measurements, we are the first to show that fine-scale variation in soil properties related to pH and water availability potentially drives the distribution of hybridizing tree species in a mixed stand. Microenvironmental variation therefore promotes local taxonomic diversity, facilitates admixture and adaptive introgression, and contributes to the resilience of forests under environmental change. Consequently, species such as white oaks should be managed and protected as a species complex rather than as pure species.


## Authors & Affiliations:

**Felix Zimmermann** (Swiss Federal Research Institute WSL, Birmensdorf, Switzerland; felix.zimmermann@wsl.ch; ORCID: 0009-0002-0762-2454)

**Oliver Reutimann** (Institute of Integrative Biology, ETH Zurich, Zurich, Switzerland; oliver.reutimann@usys.ethz.ch; ORCID: 0000-0002-5042-7518)

**Andri Baltensweiler** (Swiss Federal Research Institute WSL, Birmensdorf, Switzerland; andri.baltensweiler@wsl.ch; ORCID: 0000-0003-1933-6535)

**Lorenz Walthert** (Swiss Federal Research Institute WSL, Birmensdorf, Switzerland; lorenz.walthert@wsl.ch; ORCID: 0000-0002-1790-8563)

**Jill K. Olofsson** (Section for Forest and Landscape Ecology, Department of Geosciences and Natural Resource Management, University of Copenhagen, Denmark; jko@ign.ku.dk; ORCID: 0000-0002-9527-6573)

**Christian Rellstab** (*Corresponding Author*; Swiss Federal Research Institute WSL, Birmensdorf, Switzerland; christian.rellstab@wsl.ch; ORCID: 0000-0002-0221-5975)



## Content:

### Data:

* Here you can find the [Raw Data File](https://gitlabext.wsl.ch/felix.zimmermann/2025_eva_quercus_taxonomic_diversity/-/blob/master/Data/2025_EVA_data.xlsx?ref_type=heads).
* Here you can find the [STRUCTURE Input Files](https://gitlabext.wsl.ch/felix.zimmermann/2025_eva_quercus_taxonomic_diversity/-/tree/master/Data/STRUCTURE_Input?ref_type=heads).

### Scripts:

* Here you can find scripts for the [Variable Selection / Check for Multicollinearity](https://gitlabext.wsl.ch/felix.zimmermann/2025_eva_quercus_taxonomic_diversity/-/tree/master/Scripts/Multicollinearity?ref_type=heads).
* Here you can find scripts for the [Check for Clones / Full Siblings](https://gitlabext.wsl.ch/felix.zimmermann/2025_eva_quercus_taxonomic_diversity/-/tree/master/Scripts/Clone_Check?ref_type=heads).
* Here you can find scripts for the [Bayesian Model Averaging & Generalized Linear Models](https://gitlabext.wsl.ch/felix.zimmermann/2025_eva_quercus_taxonomic_diversity/-/tree/master/Scripts/GLM_BMA?ref_type=heads).
* Here you can find scripts for the [Multiple Regression on Distance Matrices](https://gitlabext.wsl.ch/felix.zimmermann/2025_eva_quercus_taxonomic_diversity/-/tree/master/Scripts/MRM?ref_type=heads).
* Here you can find scripts for the [Partial Redundancy Analysis](https://gitlabext.wsl.ch/felix.zimmermann/2025_eva_quercus_taxonomic_diversity/-/tree/master/Scripts/pRDA?ref_type=heads).
* Here you can find scripts for the [Q-Value Prediction with GAMLSS](https://gitlabext.wsl.ch/felix.zimmermann/2025_eva_quercus_taxonomic_diversity/-/tree/master/Scripts/Q_Value_Prediction?ref_type=heads).
* Here you can find scripts for the [Taxonomic Assignment with Support Vector Machine](https://gitlabext.wsl.ch/felix.zimmermann/2025_eva_quercus_taxonomic_diversity/-/blob/master/Scripts/SVM?ref_type=heads).

### Results:

* Here you can find results of the [Variable Selection / Check for Multicollinearity](https://gitlabext.wsl.ch/felix.zimmermann/2025_eva_quercus_taxonomic_diversity/-/tree/master/Results/Multicollinearity?ref_type=heads).
* Here you can find results of the [Check for Clones / Full Siblings](https://gitlabext.wsl.ch/felix.zimmermann/2025_eva_quercus_taxonomic_diversity/-/tree/master/Results/Clone_Check?ref_type=heads).
* Here you can find results of the [Bayesian Model Averaging & Generalized Linear Models](https://gitlabext.wsl.ch/felix.zimmermann/2025_eva_quercus_taxonomic_diversity/-/tree/master/Results/GLM_BMA?ref_type=heads).
* Here you can find results of the [Multiple Regression on Distance Matrices](https://gitlabext.wsl.ch/felix.zimmermann/2025_eva_quercus_taxonomic_diversity/-/tree/master/Results/MRM?ref_type=heads).
* Here you can find results of the [Partial Redundancy Analysis](https://gitlabext.wsl.ch/felix.zimmermann/2025_eva_quercus_taxonomic_diversity/-/tree/master/Results/pRDA?ref_type=heads).
* Here you can find results of the [Q-Value Prediction with GAMLSS](https://gitlabext.wsl.ch/felix.zimmermann/2025_eva_quercus_taxonomic_diversity/-/tree/master/Results/Q_Value_Prediction?ref_type=heads).
* Here you can find results of the [Taxonomic Assignment with Support Vector Machine](https://gitlabext.wsl.ch/felix.zimmermann/2025_eva_quercus_taxonomic_diversity/-/tree/master/Results/SVM?ref_type=heads).



## To use our code:

1. Download/clone repository.

2. Open the "2025_EVA.Rproj" file with RStudio.

3. Run scripts (you can find "Scripts" under "Files" in the "Files/Plots/Packages etc." window).
   

## Citation:

Zimmermann, F., Reutimann, O., Baltensweiler, A., Walthert, L., Olofsson, J.K. and Rellstab, C. (2025), Fine-Scale Variation in Soil Properties Promotes Local Taxonomic Diversity of Hybridizing Oak Species (*Quercus* spp.). Evol Appl, 18: e70076. [https://doi.org/10.1111/eva.70076](https://doi.org/10.1111/eva.70076)


## Link to publication:

[https://doi.org/10.1111/eva.70076](https://doi.org/10.1111/eva.70076)
